spring-cloud-kubernetes-client-config/src/main/java/org/springframework/cloud/kubernetes/client/config/reload/KubernetesClientEventBasedConfigMapChangeDetector.java

```diff
@@ -139,7 +139,7 @@ void shutdown() {
 		factory.stopAllRegisteredInformers();
 	}
 
-	private void onEvent(V1ConfigMap configMap) {
+	protected void onEvent(V1ConfigMap configMap) {
 		log.debug("onEvent configMap: " + configMap.toString());
```

Comment:
**[UNRELATED]** This is unrelated to this proposal but I wanted to share it w/ you in case you have not seen it yet. There is a LogAccessor in SF that allows you to specify message via Lambda which prevents from having to test your log statements (aka it does `if (log.isDebugEnabled()...)`). Typically I opt out of doing this test as it clutters the code ([example](https://github.com/spring-projects-experimental/spring-pulsar/blob/main/spring-pulsar/src/main/java/org/springframework/pulsar/core/CachingPulsarProducerFactory.java#L64)).

Reply:
oh wow! thank you for the `LogAccessor`, the number of times I removed `isDebugEnabled...`. We did talk with Ryan a few times about this and it's great that Spring has such a thing in place. Sorry for side stepping the comments.

Reply:
Thanks I made changes in the class, we have plenty of other spots where we can do this too.